inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongClusterController.java

```diff
@@ -228,11 +228,11 @@ public Response<Boolean> deleteNode(@PathVariable Integer id) {
         return Response.success(clusterService.deleteNode(id, LoginUserUtils.getLoginUser().getName()));
     }
 
-    @RequestMapping(value = "/cluster/node/bindTag")
+    @RequestMapping(value = "/cluster/node/bindLabel")
```

Comment:
Should the operation of binding the label be a separate operation class?

It is only associated with the file agent

Reply:
At present, the label is not bound to the file agent forcibly, and its association is also bound to the node. 

And the design of the label is not just for collection tasks

Reply:
bindNodeLabel, not bindNodeTag